evap/staff/views.py

```diff
@@ -1296,8 +1297,7 @@ def questionnaire_edit(request, questionnaire_id):
 
     if form.is_valid() and formset.is_valid():
         form.save()
-        if editable:
```

Comment:
why could you remove this line?

Reply:
the formset needs to be saved because the `allows_textanswers` field is an editable field also for forms that are otherwise not editable (see above in `make_questionnaire_edit_forms`)